frb_example/with_flutter/pubspec.yaml.release

```diff
@@ -34,7 +34,9 @@ dependencies:
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^1.0.2
+  cupertino_icons: ^1.0.5
```

Comment:
Since you have mentoined ".release files are out of sync with its counterpart", maybe we can add a phase checking them, probably in code_generator.yml. (Surely can be done in another PR)

Reply:
Well I manually synced all of them but I agree with you, it can be automated in a subsequent PR.